lua/packer/plugin_types/git.lua

```diff
@@ -173,6 +173,15 @@ git.setup = function(plugin)
     end)
   end
 
+  plugin.remote_url = function ()
+    return async(function ()
+      return await(jobs.run(fmt("%s -C %s remote get-url origin", config.exec_cmd, plugin.install_path),
+      {capture_output = true})):map_ok(function (data)
```

Comment:
Just a heads up: if `plugin.install_path` contains a space, the command will fail because `jobs.run()` uses spaces as a separator when splitting the string.

Reply:
Thanks for the heads up 👍🏾 just switched to specifying the directory using the `cwd` argument of the `job` instead